src/site/antora/modules/ROOT/pages/manual/api.adoc

```diff
@@ -220,7 +221,25 @@ link:../javadoc/log4j-api/org/apache/logging/log4j/LogManager.html[`EventLogger`
 
 xref:manual/eventlogging.adoc[Read more on event loggers...]
 
-[#feature-fluent-api]
+[#simple-logger]
+=== Simple logger
+
+Even though Log4j Core is the reference implementation of Log4j API, Log4j API itself also provides a very minimalist implementation: _Simple Logger_.
+This is a convenience for environments where either a fully-fledged logging implementation is missing, or cannot be included for other reasons.
+`SimpleLogger` is the default Log4j API implementation if no other is available in the classpath.
```

Comment:
Maybe I would say it is the "fallback" Log4j API implementation, since unlike other implementations a warning is always logged, whenever it is used.

Reply:
Fixed in 9a2815bb87b3bd0c8d7cde3ebefae095e948b049.